src/views/registration-js/attendees/fields/datetime.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/datetime.php
+ *
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Datetime
+ *
+ */
+$required   = isset( $field->required ) && 'on' === $field->required ? true : false;
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 5a3be651177c473a41ae22b03d80e5f3</sub>